CHANGES.rst

```diff
@@ -10,7 +10,8 @@ Breaking changes:
 
 New features:
 
-- *add item here*
+- Added ‘Show Toolbar’ permission.
```

Comment:
I fear that these pretty quotes could trip up zest.releaser. Could you make them normal single quotes? Or maybe `` to display it as code, or `*` to display it cursive.